packages/eslint-plugin/src/rules/no-misused-spread.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  ESLintUtils,
+  type TSESLint,
+  type TSESTree,
+} from '@typescript-eslint/utils';
+
+import { createRule } from '../util';
+
+export default createRule({
+  defaultOptions: [],
+  name: 'no-misused-spread',
+  meta: {
+    docs: {
+      description: 'Disallow spread operator on function',
+      recommended: 'stylistic',
```

Comment:
What will be your recommendation?
Should it be `strict`?

Reply:
https://typescript-eslint.io/linting/configs - this is a functional, bug-catching rule so it wouldn't be `'stylistic'`. It would be `'strict'` for now to avoid a breaking change. In each new next major version we look at which rules might be promoted to `'recommended'`.